docs/source/reference/command_line.rst

```diff
@@ -313,6 +313,43 @@ Below is a list with all available subcommands.
       list  Display a list of all available plugins.
 
 
+.. _reference:command-line:verdi-presto:
+
+``verdi presto``
+----------------
+
+.. code:: console
+
+    Usage:  [OPTIONS]
+
+      Set up a new profile as automatically as possible.
```

Comment:
This sentence sounds strange to me. Maybe...

```suggestion
      Set up a new profile without a fuss.
```

Or 
```suggestion
      Set up a new, pre-configured profile.
```

```suggestion
    Quickly set up a new profile.
```

Not quite happy with any of these, perhaps @mbercx will have the right idea.

Reply:
Also found it difficult to find a good phrasing for this. Agree that what I have currently is unsatisfactory

Reply:
Given the name of the command, perhaps something a bit light-hearted might actually work. :-)

Reply:
>  Given the name of the command, perhaps something a bit light-hearted might actually work. :-)

Yeah fair enough, how about

> Set up a new profile in a jiffy.

Not sure how well known that expression is to non-Brits though

Reply:
I _love_ `in a jiffy`! But as you mention, it probably might confuse some people. And given `presto` itself might not be known to everybody (sorry italians :it: ) we better be clear in the first sentence of the docstring.

I asked [a hippo](https://www.wordhippo.com/what-is/another-word-for/in_a_jiffy.html) for help, how about

```
Set up a profile in no time.
Set up a profile without hassle.
Set up a profile rapidly/swiftly and without hassle.
```

The last one tries to convey that it should be both quick and easy (without a fuss). But fuss or hassle might have the same problem as jiffy? Okay I'll stop painting the bikeshed now. :bike:

Reply:
It's an excellent question which clearly requires extensive debate. ^^

I think one issue in properly describing the command is that it doesn't set up a barebones profile because it _does_ check for a RabbitMQ broker, but it doesn't check for all services since for PostgreSQL that is more complicated.

However, the most important aspect of the `verdi presto` command is that it's fast. So I'm in favor of something like:

```
Set up a new profile in seconds.
```

The second aspect is that it doesn't require user input (for now). So we could add "automatically":

```
Automatically set up a new profile in seconds.
```

I'm also fine with "in no time", "without hassle", "in a jiffy". But I think "in seconds" is the easiest to understand for non-natives?

Reply:
We could also use "in a matter of seconds" or "in a few seconds". But maybe we should block a 3 hour Zoom call to discuss this at length... 😁